src/js/letters/reducers/index.js

```diff
@@ -23,16 +27,34 @@ function letters(state = initialState, action) {
       // the various error scenarios
       return set('lettersAvailable', false, state);
     case 'GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS':
+      // Create object for which options to include in post for summary letter
```

Comment:
Haha I started with that but then eslint yelled at me for `Unexpected lexical declaration in case block`. Apparently you can't declare variables inside a case block.